DataFormats/JetReco/src/Jet.cc

```diff
@@ -358,7 +358,12 @@ Jet::Constituents Jet::getJetConstituents () const {
 std::vector<const Candidate*> Jet::getJetConstituentsQuick () const {
   std::vector<const Candidate*> result;
   int nDaughters = numberOfDaughters();
-  for (int i = 0; i < nDaughters; ++i) { 
+  for (int i = 0; i < nDaughters; ++i) {
+    if(!(this->sourceCandidatePtr(i).isNonnull() && this->sourceCandidatePtr(i).isAvailable()))
+    {
+        std::cerr << "Bad pointer to jet constituent found.  Check for possible dropped particle." << std::endl;
```

Comment:
std stream (cerr) is not allowed.
Please switch to LogWarning.
If these conditions happen in more than ~1% of events, this should become a LogInfo or even LogTrace.